featuretools/feature_base/feature_base.py

```diff
@@ -82,7 +82,7 @@ def get_name(self):
             self._name = self.generate_name()
         return self._name
 
-    def get_names(self):
+    def __get_names(self):
```

Comment:
If we keep this method, we should use a single `_` in the name: `def _get_names(self):`